man/manual-premium.md

```diff
@@ -749,29 +773,35 @@ To get gcc compatible output you can use --template=gcc:
 The output will look like this:
 
     Checking samples/arrayIndexOutOfBounds/bad.c ...
-    samples/arrayIndexOutOfBounds/bad.c:6:6: warning: Array 'a[2]' accessed at index 2, which is out of bounds. [arrayIndexOutOfBounds]
+    samples/arrayIndexOutOfBounds/bad.c:6:6: warning: Array
+    'a[2]' accessed at index 2, which is out of bounds. [arrayIndexOutOfBounds]
     a[2] = 0;
       ^
 
 ## User defined output format (single line)
 
-You can write your own pattern. For instance, to get warning messages that are formatted like traditional gcc, then the following format can be used:
+You can write your own pattern. For instance:
 
-    cppcheck --template="{file}:{line}: {severity}: {message}" samples/arrayIndexOutOfBounds/bad.c
+    cppcheck --template="{file}:{line}:{column}: {severity}:
```

Comment:
can we try to rewrite this somehow, this can't be copy pasted directly as-is by a user.